esx_service/vmdk_ops_test.py

```diff
@@ -1008,18 +1045,19 @@ def test_vmdkops_on_tenant_vm(self):
         error_info = vmdk_ops.executeRequest(vm1_uuid, self.vm1_name, self.vm1_config_path, auth.CMD_CREATE, self.tenant1_vol2_name, opts)
         self.assertEqual({u'Error': 'volume size exceeds the max volume size limit'}, error_info)
 
+        # create a volume with 500MB
         opts={u'size': u'500MB', u'fstype': u'ext4'}
         error_info = vmdk_ops.executeRequest(vm1_uuid, self.vm1_name, self.vm1_config_path, auth.CMD_CREATE, self.tenant1_vol2_name, opts)
         self.assertEqual(None, error_info)
 
-        # create a volume with 500MB, and total_storeage used by this tenant will exceed volume_totalsize
+        # create another volume with 500MB, and total_storeage used by this tenant will exceed volume_totalsize
         opts={u'size': u'500mb', u'fstype': u'ext4'}
         error_info = vmdk_ops.executeRequest(vm1_uuid, self.vm1_name, self.vm1_config_path, auth.CMD_CREATE, self.tenant1_vol3_name, opts)
         self.assertEqual({u'Error': 'The total volume size exceeds the usage quota'}, error_info)
 
         # set allow_create to False
         error_info = auth_api._tenant_access_set(name=self.tenant1_name,
-                                                 datastore=self.datastore_name,
+                                                 datastore=auth_data_const.VM_DS,
```

Comment:
This should be disallowed, default DS privileges shouldn't be modifiable to disallow create and ls. There isn't any use case for it (yes?).